composeApp/src/commonMain/kotlin/sp/bvantur/tasky/home/data/HomeRepositoryImpl.kt

```diff
@@ -57,4 +59,19 @@ class HomeRepositoryImpl(
     override suspend fun logoutUser(): TaskyEmptyResult<TaskyError> = localDataSource.clearLocalData().onSuccess {
         remoteDataSource.logoutUser()
     }
+
+    override suspend fun deleteEventById(id: String): TaskyEmptyResult<TaskyError> {
+        val localResult = localDataSource.deleteEventById(id)
+
+        if (localResult.isError()) {
+            // TODO handle database error
```

Comment:
What do you usually handle error in this case? When it fails on removal? User message that it can be removed? Or just fallback to scheduling a worker to try to remove from database and remote API later on?